wagtail/embeds/oembed_providers.py

```diff
@@ -199,11 +197,11 @@
 geograph_gg = {
     "endpoint": "http://www.geograph.org.gg/api/oembed",
     "urls": [
-        r"^https?://(?:[-\w]+\.)?geograph\.org\.gg/.+$",
-        r"^https?://(?:[-\w]+\.)?geograph\.org\.je/.+$",
-        r"^https?://channel-islands\.geograph\.org/.+$",
-        r"^https?://channel-islands\.geographs\.org/.+$",
-        r"^https?://(?:[-\w]+\.)?channel\.geographs\.org/.+$",
+        r"^https?://.*\.geograph\.org\.gg/.*$",
```

Comment:
Question: Could you explain this change and why it's needed? This pattern looks much broader than the previous.